.github/workflows/deploy-vc-api.yml

```diff
@@ -0,0 +1,62 @@
+name: Build and deploy VC-API
+
+on: [push, workflow_dispatch]
+
+jobs:
+  cancel-previous:
+    name: 'Cancel Previous Runs'
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    steps:
+      - uses: styfle/cancel-workflow-action@0.9.1
+        with:
+          access_token: ${{ github.token }}
+
+  unique_id:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate unique id
+        id: unique_id
+        run: echo "id=$(uuidgen) >> $GITHUB_OUTPUT"
+    outputs:
+      unique_id: ${{ steps.unique_id.outputs.id }}
+
+  install-build-and-push:
+    runs-on: ubuntu-latest
+    needs: [cancel-previous, unique_id]
+    steps:
+      - name: Check out vc-api-owf
+        run: |
+          git clone https://github.com/energywebfoundation/vc-api-owf
+
+      - name: Configure AWS credentials IDC
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-access-key-id: ${{ secrets.DEV_AWS_ACCESS_KEY_ID  }}
+          aws-secret-access-key: ${{ secrets.DEV_AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_REGION }}
+
+      - name: Login to Amazon ECR
+        id: login-ecr-ssi
+        uses: aws-actions/amazon-ecr-login@v2
+
+      - name: Build Docker image
+        working-directory: ./vc-api-owf
+        env:
+          ECR_REGISTRY: ${{ steps.login-ecr-ssi.outputs.registry }}
+          AWS_ECR_REPOSITORY_SSI: ${{ secrets.DEV_ECR_REPOSITORY }}
+        run: |
+          echo $ECR_REGISTRY
+          docker build \
+            -t $ECR_REGISTRY/$AWS_ECR_REPOSITORY_SSI \
+            -f apps/vc-api/Dockerfile .
+
+#      - name: Push Docker image
+#        working-directory: ./vc-api-owf
+#        run: |
+#          docker push \
+#            $ECR_REGISTRY/$AWS_ECR_REPOSITORY_SSI
```

Comment:
Is this just commented for now while testing the other steps?

Reply:
Yes, I just don't want image to be pushed to the repository when testing this.